x-pack/test/security_solution_api_integration/test_suites/investigation/saved_objects/trial_license_complete_tier/helpers.ts

```diff
@@ -38,3 +42,93 @@ export const createBasicTimelineTemplate = async (
         timelineType: TimelineTypeEnum.template,
       },
     });
+
+export const deleteAllNotes = async (es: Client): Promise<void> => {
+  await es.deleteByQuery({
+    index: SECURITY_SOLUTION_SAVED_OBJECT_INDEX,
+    q: `type:${noteSavedObjectType}`,
+    wait_for_completion: true,
+    refresh: true,
+    body: {},
+  });
+};
+
+export const createNoteAssociatedWithDocumentOnly = async (
+  supertest: SuperTest.Agent,
+  documentId: string,
+  noteText: string
+) =>
+  await supertest
+    .patch(NOTE_URL)
+    .set('kbn-xsrf', 'true')
+    .send({
+      note: {
```

Comment:
💅 You could use the actual request body types here and in the cases below, so that it will be easier to refactor these tests once we are changing the request body types.

Reply:
good point, fixed [here](https://github.com/elastic/kibana/pull/196225/commits/fcd19df73e926148dce5a070d467bd8378184a63)